api/app/routes.py

```diff
@@ -29,6 +29,7 @@
 rest_api.add_resource(content_api.CountryContentAPI,
                       '/api/v1/content/countries')
 rest_api.add_resource(files_api.FileUploadAPI, '/api/v1/file')
+rest_api.add_resource(files_api.FileAPIImproved, '/api/v1/file')
```

Comment:
URIs (Uniform Resource Identifiers) need to be different. So set this to `/api/v1/fileimproved` or something

Reply:
Hi @amritpurshotam ! I Think i'm done with the suggested updates. Hope there is no longer git mistakes ? :) ! Is another PR as suggested so far still necessary ?. Thanks a bunch for your help, i really appreciate !